### ISSUE TITLE
TypeError when setting Window.size

### ISSUE THREAD

**Issue by animalmutch** [kivy/kivy#7616]
**Software Versions**
* Python: 3.7.3
* OS: Raspberry Pi OS Lite (32-bit)
* Kivy: 2.0.0
* Kivy installation method: https://gist.github.com/animalmutch/8a6a23fdf62b0f49d8409d513e314ffe

**Describe the bug**
When attempting to set Window.size, a TypeError is raised
https://gist.github.com/animalmutch/a9bca2ed65368ee7de35407836d3d077

**Expected behavior**
That I am able to set Window.size (as has always been the case until very recently)

**To Reproduce**
Just import kivy.core.window.Window into any otherwise valid piece of code, and set Window.size. Done here with one of the standard Kivy examples:

https://gist.github.com/animalmutch/c856a87d754a2126c60ad6de4165d77f

**Comment by p0lygun** [comment]
@animalmutch I am unable to reproduce error 
```py
from kivy.app import App
from kivy.uix.label import Label
from kivy.core.window import Window
Window.size = (800, 480)


class Temp(App):

    def build(self):
        return Label(text="working")


Temp().run()
```
works fine

**Comment by p0lygun** [comment]
i see that u are using rasp os

**Comment by animalmutch** [comment]
@p0lygun Yes, that's right. I'm using the lite version, without the desktop environment, so have built Kivy from source I think according to the instructions in the documentation. I've given a link to the bash commands I used. Perhaps I'm doing something wrong in that process?

**Comment by p0lygun** [comment]
@animalmutch hmm could be i'll install from source on my rasp as soon as i get a bit of free time and report my findings

**Comment by animalmutch** [comment]
Thanks! :)

**Comment by Julian-O** [comment]
Is this still a problem with Kivy 2.2.1 (and up to date dependencies)?

**Comment by animalmutch** [comment]
Afraid I've not had a chance to try, and probably won't get to any time in the near future. In the end, as we only needed to set the window size in a test environment on windows, we just worked around it by detecting the OS.
